core/LocalVariable.cc

```diff
@@ -24,7 +24,7 @@ bool LocalVariable::isSyntheticTemporary(const GlobalState &gs) const {
            _name == Names::arrayTemp() || _name == Names::rescueTemp() || _name == Names::exceptionValue() ||
            _name == Names::exceptionClassTemp() || _name == Names::gotoDeadTemp() || _name == Names::isaCheckTemp() ||
            _name == Names::throwAwayTemp() || _name == Names::castTemp() || _name == Names::finalReturn() ||
-           _name == Names::cfgAlias() || _name == Names::magic();
+           _name == Names::cfgAlias() || _name == Names::magic() || _name == Names::argPresent();
```

Comment:
What does this do?

Reply:
It's used in dealias to look through assignments. Given how this variable is used, it doesn't seem like it's useful to have it in the list. Should I remove it?

Reply:
I don't really know what this list does. I'm happy either way, but I would love to learn more about what the considerations are. Maybe we could figure them out and leave a comment on the method?

Reply:
I removed it and the tests were fine with that, so I'll just leave the list alone.